packages/back-end/src/services/stripe.ts

```diff
@@ -50,9 +67,19 @@ export async function updateSubscriptionInDb(
       cancel_at_period_end: subscription.cancel_at_period_end,
       planNickname: item?.plan?.nickname,
       priceId: item?.price?.id,
+      hasPaymentMethod: hasPaymentMethod,
     },
     priceId: item?.price?.id,
   });
+
+  // update free trial status
+  if (subscription.status === "trialing") {
+    if (org && !org.freeTrialDate) {
+      await updateOrganization(org.id, {
+        freeTrialDate: new Date(),
+      });
```

Comment:
Right above this is already updating the organization object. Can we combine these so it's only one db update call?

Reply:
yes, I'd meant to do this earlier... Combined the queries now.